src/context.ts

```diff
@@ -700,6 +711,21 @@ export class Context implements RenamedUpdate {
         customEmojiKept: string[];
         /** Custom emoji removed from this user's reaction */
         customEmojiRemoved: string[];
+        /**
+         * `true` if a paid reaction is currently present in this user's
+         * reaction, and `false` otherwise
+         */
+        paid: boolean;
+        /**
+         * `true` if a paid reaction was newly added to this user's reaction,
+         * and `false` otherwise
+         */
+        paidAdded: boolean;
+        /**
+         * `true` if a paid reaction was removed from this user's reaction, and
```

Comment:
Feel free to ignore.

```suggestion
         * `true` if a paid reaction was just added to this user's reaction,
         * and `false` otherwise
         */
        paidAdded: boolean;
        /**
         * `true` if a paid reaction was just removed from this user's reaction, and
```

Reply:
I will. It is inconsistent with the wording above for emoji reactions and custom emoji reactions.